lib/vorpal/config_builder.rb

```diff
@@ -90,7 +90,7 @@ def build_db_class
       # Module#parent comes from 'active_support/core_ext/module/introspection'
       parent_module = @domain_class.parent
 
-      return parent_module.const_get(db_class_name) if parent_module.const_defined?(db_class_name)
+      return parent_module.const_get(db_class_name) if Object.const_defined?(fully_qualified_db_class_name)
```

Comment:
Perhaps a more elegant change would have been to use `parent_module.const_defined?(db_class_name, false)` which, according to [the ruby docs](http://ruby-doc.org/core-2.2.0/Module.html#method-i-const_defined-3F), will not check ancestors. This would have avoided the need for the `fully_qualified_db_class_name` method and the duplication of the default naming prefix.

I will not block the merge request because of this, but will probably clean it up later.